lib/cWarnings.mli

```diff
@@ -122,6 +122,7 @@ module CoreCategories : sig
   val parsing : category
   val pedantic : category
   val records : category
+  val rewrite_rules : category
```

Comment:
Should the lsit in `doc/sphinx/proof-engine/vernacular-commands.rst` be updated?

Reply:
yes, changed

Reply:
We may want to put a comment in that mli file to remember about that list in the doc next time.

Reply:
There's a comment in the .ml, but the list is big enough that it doesn't appear in the diff context